### PR TITLE
Make BDA module

### PR DESCRIPTION
**Pull request by plaplant**
This PR moves a lot of the X-engine start-up scripting into a new `bda.py` module inside of the python package. We are also now putting the BDA baseline info into redis, rather than writing out temporary files on each of the x-boxes. This should streamline startup and make the information more reliable and transparent to other potential consumers of this information (e.g., M&C).

### CODE REVIEW

**Comment by plaplant** [comment]
Commit e441182 has been confirmed to work on site. (The commit afterward only removes commented out code from a python script.)

File: control_software/scripts/hera_catcher_up.py [hunk 1]
```diff
@@ -67,17 +67,6 @@ def run_on_hosts(hosts, cmd, user=None, wait=True):
...
-
-# Upload config file location
-# NOTE: This has to come before template generation!
-# Generate the BDA config file and upload to redis
-if not args.nobda:
-    print('Create configuration file')
-    run_on_hosts([args.host], python_source_cmd + bda_config_cmd + ['-c','-r', '/tmp/bdaconfig.txt'], wait=True)
-    os.system('scp "%s:%s" "%s" ' % ('hera-sn1', '/tmp/bdaconfig.txt','/tmp/bdaconfig.txt') )
-    
 time.sleep(10)
```
Comment (AaronParsons) [inline control_software/scripts/hera_catcher_up.py:70]: Is this sleep still necessary?

File: control_software/src/paper_gpu/gpu.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022 The HERA Collaboration
+# Licensed under the 2-clause BSD License
+
+pass
```
Comment (AaronParsons) [inline control_software/src/paper_gpu/gpu.py:5]: Maybe delete this module for now

**Review by AaronParsons — CHANGES_REQUESTED** [review]
Looks good. Just 2 ideas: delete gpu.py until we need it, and take a look at that extra 10s sleep. If needed, that's fine. Just wasn't sure.

File: control_software/scripts/hera_catcher_up.py (hunk 1 above)
Comment (plaplant) [reply control_software/scripts/hera_catcher_up.py:70]: I tried to run a1759f0 (where this sleep was removed), and it didn't work. I added it back in with 2b1fdd7 and that worked on site.

File: control_software/src/paper_gpu/gpu.py (hunk 2 above)
Comment (plaplant) [reply control_software/src/paper_gpu/gpu.py]: Good idea, removed.

**Comment by plaplant** [comment]
@AaronParsons thanks for the review! The latest version has been shown to work on site.

**Review by AaronParsons — APPROVED** [review]

**Pull request merged**